public/src/components/channelManagement/bannerTests/bannerTestEditor.tsx

```diff
@@ -157,6 +165,19 @@ const BannerTestEditor: React.FC<ValidatedTestEditorProps<BannerTest>> = ({
   if (test) {
     return (
       <div className={classes.container}>
+        <div className={classes.sectionContainer}>
+          <Typography variant={'h3'} className={classes.sectionHeader}>
+            Campaign
```

Comment:
I'm worried about pushing important things down the page - and having an h3 with the dropdown below it takes up a lot of room! What do you think about having a label + dropdown horizontally instead?

Reply:
Campaigns section moved down the page to follow the variants, and variants split editor, sections